tests/pgsql-split-brain-isolated-standby/run.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+docker-compose down -v && docker-compose build && docker-compose up -d pgmaster pgslave1 pgslave3
```

Comment:
What's the point of `NO_CLEANUP` if you are still going to drop the environment?
I think we should move step with all pre-requirements (clean-up, build and so on) to the main runner.

Reply:
might be